include/vrv/mrest.h

```diff
@@ -22,10 +23,10 @@ namespace vrv {
  * This class models the MEI <mRest> element.
  */
 class MRest : public LayerElement,
+              public DurationInterface,
               public PositionInterface,
               public AttColor,
               public AttCue,
-              public AttFermataPresent,
```

Comment:
Why do you want to remove this?

Reply:
AttFermataPresent is included in DurationInterface